### PR TITLE
Feature/user timeline

### PR DESCRIPTION
**Pull request by lee-benjamin**

### CODE REVIEW

File: src/js/twitter.js [hunk 1]
```diff
@@ -20,8 +21,8 @@ export const getTwitterLink = (tweet) => {
   return urlPrefix + tweet.id;
 }
 
-export const getHomeTimeline = (successCallback, failureCallback) => {
-  const url = "http://localhost:8080/api/1.0/twitter/timeline"
+export const getTimeline = (endpoint, successCallback, failureCallback) => {
```
Comment (kev-chan) [inline src/js/twitter.js]: let's have explicit endpoints such as your getHomeTimeline before the code changes here. Create another explicit endpoint for user timeline

File: src/styles/sass/style.scss [hunk 2]
```diff
@@ -11,57 +13,90 @@ $left-margin: 10px;
...
+
+  .TimelineHeader {
+    margin-bottom: 50px;
+    text-align: center;
+  }
+
+  .Timeline {
+    margin: $margin;
+    border: 1px black solid;
+    .Tweet {
```
Comment (kev-chan) [inline src/styles/sass/style.scss]: add more white space between classes, makes it more readable

Comment (lee-benjamin) [reply src/styles/sass/style.scss]: ok

File: src/js/twitter.js (hunk 1 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: ok

File: src/js/components.js [hunk 3]
```diff
@@ -1,35 +1,73 @@
...
+
+  function failureCallback() {
+    ReactDOM.render(e(ReactContainer, {isServerError: true}),
+      document.getElementById("root"));
+  }
+
+  const getBothTimelines = (successCallback, failureCallback) => {
+    const url = "http://localhost:8080/api/1.0/twitter/timeline/";
+    let promises = [url+"home", url+"user"].map(url => fetch(url).then(resp => resp.json()));
+    Promise.all(promises)
```
Comment (kev-chan) [inline src/js/components.js]: have the created endpoints be reusable and use those here as opposed to recreating the urls again

Comment (lee-benjamin) [reply src/js/components.js]: done

**Pull request merged**